lib/Page.js

```diff
@@ -728,8 +729,11 @@ class Page extends EventEmitter {
     return this.mainFrame().title();
   }
 
+  _closed() { this.emit(Page.Events.Closed, 'page closed'); }
```

Comment:
the 'closed' event is not needed any more since we have target events; they should cover everything